source/opt/function.h

```diff
@@ -148,7 +148,14 @@ class Function {
   BasicBlock* InsertBasicBlockBefore(std::unique_ptr<BasicBlock>&& new_block,
                                      BasicBlock* position);
 
-  // Return true if the function calls itself either directly or indirectly.
+  // If this function has only one OpReturnValue instruction,
+  // then returns the value id. Otherwise, returns 0.
```

Comment:
As mentioned above, please move this into the transformation.  Then the comment should be described as a precondition:

- Requires that the function contains at most one OpReturnValue instruction.  Returns the id associated with this instruction if present, and 0 otherwise.

Reply:
Done.